app/frontend/src/service/user.ts

```diff
@@ -231,3 +232,15 @@ export async function completeSignup({
   const jailed = res.getJailed();
   return { token: sessionToken, jailed };
 }
+
+/**
+ * Logout user using a session token
+ */
+export async function logout(sessionToken: string) {
+  const req = new DeAuthReq();
+  req.setToken(sessionToken);
+
+  await client.auth.deauthenticate(req);
+
+  return null;
```

Comment:
No need to return anything, for gRPC you can assume it's successful if it doesn't throw an error in the client call. It will throw for timeouts etc.